docs/extensions/VisualExtensions.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Composition Visual Attached Properties
+title: Composition Visual Extension 
 author: nmetulev
-description: The Composition Visual Attached Properties allow Composition Visual Properties to be modified directly in XAML
+description: The Composition Visual Attached Properties Extension allow Composition Visual Properties to be modified directly in XAML

```

Comment:
"Properties Extension allows" - if there's only one of them
"Property Extensions allow" - if there's more than one extension